pulp_smash/utils.py

```diff
@@ -9,6 +9,7 @@
 from packaging.version import Version
 from pulp_smash.constants import REPOSITORY_PATH, USER_PATH
 from time import sleep
+from urlparse import urljoin
```

Comment:
I will make it so.

Relative paths with leading slashes are just one demonstration of how the behavior is different. Here is another where string addition would produce an incorrect result:

```
>>> urljoin('http://foo.com', 'a/b/c')
'http://foo.com/a/b/c'
```